### DISCUSSION TITLE
是否可以搞一个自托管gitea镜像

### DISCUSSION THREAD

**Discussion by uubulb** [nezhahq/nezhahq.github.io#78] [General]
选择gitea是因为比gitlab稍微轻量一点，也有官方的golang sdk支持，之后可以整合进自动更新模块中，方便国内用户自动更新agent

**Comment by naiba** [comment]
不考虑 没人维护

**Reply by uubulb** [reply]
那gitee呢

**Comment by naiba** [comment]
Gitee有了 只是没同步release

**Comment by naiba** [comment]
可以想办法同步一下

**Reply by uubulb** [reply]
如果要把agent自动更新也做了，也要改一下 [gitee.com/openeuler/go-gitee](gitee.com/openeuler/go-gitee)，原版release那里的asset部分没处理好

**Reply by naiba** [reply]
晚点我到gitee fork一下

**Reply by uubulb** [reply]
> 晚点我到gitee fork一下

也可以在nezhahq开个新仓库，我pr过去

**Reply by uubulb** [reply]
> 晚点我到gitee fork一下

https://github.com/nezhahq/agent/blob/main/cmd/agent/main.go#L126
原本那个超时时间不太够了，至少我这里到gitee不止5秒，一定会超时，改成30基本不会超时

**Reply by naiba** [reply]
> > 晚点我到gitee fork一下
> 
> 
> 
> https://github.com/nezhahq/agent/blob/main/cmd/agent/main.go#L126
> 
> 原本那个超时时间不太够了，至少我这里到gitee不止5秒，一定会超时，改成30基本不会超时

可以

**Reply by uubulb** [reply]
> > > 晚点我到gitee fork一下
> > 
> > 
> > https://github.com/nezhahq/agent/blob/main/cmd/agent/main.go#L126
> > 原本那个超时时间不太够了，至少我这里到gitee不止5秒，一定会超时，改成30基本不会超时
> 
> 可以

现在有时间搞吗

**Reply by naiba** [reply]
https://gitee.com/naibahq/go-gitee

**Reply by uubulb** [reply]
> https://gitee.com/naibahq/go-gitee

已经提pr了
